src/main/java/com/dropbox/core/android/AuthActivity.java

```diff
@@ -192,28 +215,42 @@ static void setAuthParams(String appKey, String desiredUid,
      */
     static void setAuthParams(String appKey, String desiredUid,
                               String[] alreadyAuthedUids, String webHost, String apiType) {
-        setAuthParams(appKey, desiredUid, alreadyAuthedUids, null, null, null);
+        setAuthParams(appKey, desiredUid, alreadyAuthedUids, null, null, null, null, null, null);
     }
 
     /**
      * Set static authentication parameters
      */
     static void setAuthParams(String appKey, String desiredUid,
                               String[] alreadyAuthedUids, String sessionId) {
-        setAuthParams(appKey, desiredUid, alreadyAuthedUids, sessionId, null, null);
+        setAuthParams(appKey, desiredUid, alreadyAuthedUids, sessionId, null, null, null, null,
+            null);
     }
 
     /**
      * Set static authentication parameters
      */
     static void setAuthParams(String appKey, String desiredUid,
-                              String[] alreadyAuthedUids, String sessionId, String webHost, String apiType) {
```

Comment:
does this really need to take both `host` and `webHost`?  It's confusing to me which takes priority.

If it's really needed, please add parameter documentation